Workbooks/ServiceBus/AtResource/ResourceInsights.workbook

```diff
@@ -0,0 +1,2500 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "crossComponentResources": [
```

Comment:
We can delete this crossComponentResources part since we don't need subscription dropdown